knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml

```diff
@@ -704,15 +704,22 @@ metadata:
   labels:
     app.kubernetes.io/version: v1.9
     knative.dev/crd-install: "true"
-  name: consumers.internal.kafka.eventing.knative.dev
+  name: consumergroups.internal.kafka.eventing.knative.dev
```

Comment:
@pierDipi @matzew is it safe to land this on 1.30 branch? This is produced by `make generated-files` and flagged by validation check as missing. But I'd rather double check.

Reply:
OK it seems like an ordering swap to me. I hope that's fine then. :)

Reply:
Yes

Reply:
Thanks for confirming that.